runtime/src/core/runner.js

```diff
@@ -112,22 +53,6 @@ export class GrainRunner {
     await located.start();
   }
 
-  // [HACK] Temporarily used while we transition to Grain-based runtime
-  _makeGrainString(v) {
-    let buf = this.encoder.encode(v);
-    let userPtr = this.managedMemory.malloc(4 * 2 + ((v.length - 1) / 4 + 1));
-    let ptr = userPtr / 4;
-    let view = this.managedMemory.view;
-    view[ptr] = GRAIN_STRING_HEAP_TAG;
-    view[ptr + 1] = v.length;
-    let byteView = this.managedMemory.u8view;
-    for (let i = 0; i < buf.length; ++i) {
-      byteView[i + ptr * 4 + 8] = buf[i];
-    }
-    return userPtr;
-  }
-
-  // [HACK] Temporarily used while we transition to Grain-based runtime
```

Comment:
Why is this comment removed? Does it no longer apply to `grainValueToString`?

Reply:
This particular function was never really a hack... I think the comment was by mistake (since it looks like it was just copy-pasted onto everything).